apps/condo/domains/banking/schema/BankCategory.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Generated by `createschema banking.BankCategory 'name:Text;'`
+ */
+
+const { Text, Relationship, Integer, Select, Checkbox, DateTimeUtc, CalendarDay, Decimal, Password, File, Url } = require('@keystonejs/fields')
+const { Json, LocalizedText } = require('@condo/keystone/fields')
+const { GQLListSchema } = require('@condo/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@condo/keystone/plugins')
+const access = require('@condo/domains/banking/access/BankCategory')
+
+
+const BankCategory = new GQLListSchema('BankCategory', {
+    schemaDoc: 'Expenses category, that will be associated with transactions and contractors. Logically organized as a tree, which is stored in Nested Set format. To act as Nested Set, a one common technical root node should be presented, that will contain logically top-most categories as direct descendants. Categories are not intended to be changed frequently. After changes of the categories tree topology, index of Nested Set should be regenerated by manually calling buildNestedSetIndex function',
+    fields: {
+
+        name: {
+            schemaDoc: 'Key for i18n',
+            type: LocalizedText,
+            isRequired: true,
+            template: 'banking.category.*.name',
+        },
+
+        // TODO(antonal): extract `parent`, `left` and `right`, `sortOrder` to plugin and name it as 'nestedSet`
+        parent: {
```

Comment:
You need to check. Probably we don't need a tree structure here.

Reply:
Forked Nested Set tree into separate branch for future needs